### PR TITLE
Resource Identifier metadata

### PR DESCRIPTION
**Pull request by zachdaniel**
Attempting to address #202

### CODE REVIEW

File: lib/ja_serializer/builder/resource_identifier.ex [hunk 1]
```diff
@@ -48,4 +49,21 @@ defmodule JaSerializer.Builder.ResourceIdentifier do
...
+  when is_atom(identifier_metadata_name) do
+    apply(serializer, identifier_metadata_name, [source_data, destination_data, conn])
+  end
+
+  defp get_meta(
+    destination_data,
+    %{data: source_data, conn: conn},
+    %{identifer_meta: identifier_metadata_func})
+  when is_function(identifier_metadata_func) do
+   identifier_metadata_func.(source_data, destination_data, conn)
```
Comment (alanpeabody) [inline lib/ja_serializer/builder/resource_identifier.ex]: Hice, does this work? I have had some trouble getting functions working when passed to the DSL.

**Comment by zachdaniel** [comment]
@alanpeabody The function thing didn't work, since the DSL macro needs to unquote it.  I thought it did for some reason.  Good catch :D

**Comment by zachdaniel** [comment]
@alanpeabody Hey guys!  Any thoughts on this?

**Comment by alanpeabody** [comment]
Hey @zachdaniel honestly is looks great, however I am hesitant to merge it until the other meta data is supported and I can see the DSL/Implementation in full. I know it probably isn't something you need but adding it isn't my top priority ATM either, though I would like to get around to it sometime.

**Comment by zachdaniel** [comment]
Do you mean metadata for each included item?  If so, I don't know if there is a good way to do it, because it would have to override the data that is generated by the destination serializer's metadata function.  So having meta in the includes is already supported.

**Comment by asummers** [comment]
Hi @alanpeabody Just curious if there's been any movement on this.

**Comment by suhrawardi** [comment]
Any news on this one? Would love to get it merged in as well... ! 😃

**Pull request closed without merging**